components/Table/Table.js

```diff
@@ -95,12 +113,184 @@ export class Table {
     this.tableHeader.append(tableHeaderRow);
   }
 
+  renderDropDownMenu(menuBtn, isSortable, cell) {
+    const dropDownMenu = document.createElement('div');
+    dropDownMenu.classList.add('drop-down__menu');
+    this.menuDropDownItems = this.menuConfig
+      .filter((elConfig) => {
+        if (
+          this.callbacksMap.sortCallback ||
+          this.callbacksMap.filterCallback
+        ) {
+          const menuElSort =
+            elConfig.key === MenuElMap.SORT_BY_ASC ||
+            elConfig.key === MenuElMap.SORT_BY_DESC;
+          if (!isSortable && menuElSort) {
+            return;
+          }
+        }
+        return elConfig != null;
```

Comment:
We could just use `Boolean(elConfig)` or `!!elConfig`
But why do we have to check it? Do we have undefined or nulls in the menu config array?